src/Search/Search.js

```diff
@@ -26,6 +26,12 @@ const Search = ({
     onChange('')
   }
 
+  useEffect(() => {
+    document.addEventListener('clearsearchterminput', onClearClick)
+    return () =>
+      document.removeEventListener('clearsearchterminput', onClearClick)
+  }, [])
```

Comment:
It's hard to track such code/events. In this case, component should accept the reference where the `onClearClick` will be assigned to